tide_media.install

```diff
@@ -643,3 +646,52 @@ function tide_media_update_8039() {
     }
   }
 }
+
+/**
+ * Enables entity_usage module.
+ */
+function tide_media_update_8040() {
+  $tideOperation = new TideOperation();
+  $tideOperation->enableEntityUsage();
+  if (\Drupal::moduleHandler()->moduleExists('entity_usage')) {
+    $config_factory = \Drupal::configFactory();
+    $config = $config_factory->getEditable('entity_usage.settings');
+    $enabled_plugins = [
```

Comment:
@MdNadimHossain, do these config steps need to happen during hook_install for new sites? Can it ship as config or be abstracted out into TideOperation?

Reply:
My first thought was yes and tried doing that. For some wired reason, it fails in module build, but passes in project build for example in content-vic. See the module build in CI [here](https://app.circleci.com/pipelines/github/dpc-sdp/tide_media/180/workflows/853d69ba-37d8-4bcb-b85c-39ba4bbf20ae/jobs/1218) and related commit [here](https://github.com/dpc-sdp/tide_media/pull/69/commits/3c6f3e8b4fda1e92f4b6d8b3fb6252ad2913c421) 
Hmm now that you pointed out, the right way is to ship it in config I guess, so that builds in tide_media install and update the settings in the update hook for the existing site. Let me add the config in the install directory and see if it passes the build

Reply:
bummer, it won't work putting it into the configs as it will trigger error like `PreExistingConfigException`
will try one more time, just to putting it in the install hook